### PR TITLE
[sparkle] - chore: general polish

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR refactors several UI components in Sparkle using class-variance-authority (CVA) for better styling organization and consistency. Key changes include:
- Avatar component refactored to use CVA for variant management and cleaner styling 
- ContentMessage component updated with improved layout and standardized story structure 
- Banner component removed as it was unused 
- General cleanup of component classes and message styling

The styling changes were taken from https://github.com/dust-tt/dust/pull/8546/

**References:**
- https://github.com/dust-tt/dust/pull/8546/

## Risk

Breaking UI

## Deploy Plan

- Publish sparkle alpha
- Use sparkle alpha in front
- Publish sparkle
- Deploy front with new sparkle version

### CODE REVIEW

File: sparkle/src/stories/ContentMessage.stories.tsx [hunk 1]
```diff
@@ -1,58 +1,98 @@
...
-  <div className="s-flex s-flex-col s-gap-4">
-    <h2>Size "md"</h2>
-    <ContentMessage title="This is a title">
-      This is a message. It can be multiple lines long.
-    </ContentMessage>
-    <ContentMessage title="This is a title" variant="pink">
-      This is a message. It can be multiple lines long.
-    </ContentMessage>
-    <ContentMessage title="This is a cat" variant="purple" icon={HeartIcon}>
-      This is a Soupinou. It is cute.
```
Comment (PopDaph) [inline sparkle/src/stories/ContentMessage.stories.tsx:23]: 😢

File: sparkle/src/stories/ContentMessage.stories.tsx [hunk 2]
```diff
@@ -1,58 +1,98 @@
...
+    title: {
+      control: "text",
+      description: "Title of the message",
+    },
+    children: {
+      control: "text",
+      description: "Content of the message",
+    },
+    variant: {
+      options: ["default", "slate", "emerald", "pink", "purple"],
```
Comment (PopDaph) [inline sparkle/src/stories/ContentMessage.stories.tsx:31]: We can't fetch the list of variants from the component?

File: sparkle/src/components/Avatar.tsx [hunk 3]
```diff
@@ -1,93 +1,176 @@
...
-  lg: "s-text-3xl",
-  xl: "s-text-5xl",
-  xxl: "s-text-7xl",
-  auto: "s-text-xl",
-};
+}
 
 export function Avatar({
-  size = "md",
+  size,
```
Comment (PopDaph) [inline sparkle/src/components/Avatar.tsx:173]: Why forcing size and classname?

**Review by PopDaph — APPROVED** [review]
🙌🏻

File: sparkle/src/components/Avatar.tsx (hunk 3 above)
Comment (JulesBelveze) [reply sparkle/src/components/Avatar.tsx:173]: They are actually optional and size gets a default value in the cva

**Pull request merged**